data/human/human missions.txt

```diff
@@ -5428,3 +5428,120 @@ mission "Deal Change 2"
+			
+			label "can't take package"
+			`	"I'm sorry, but we can't take this package," they say.`
+			choice
+				`	"Is something wrong?"`
+					goto misdirected
+				`	"What do you mean? Surely you can."`
+			`	"Again, I'm sorry, but we can't send the package," they reiterate. "It's got an incorrect address.`
+			label misdirected
+			`	"This package is addressed to 2081 Amethyst Avenue, Mercury City, <origin>. Mercury City is <planet>'s capital. Y'know, that planet out east in Algol. Syndicate space. Not a place on <origin>. The package was posted to the wrong planet."`
+			`	"Oh," you say.`
+			`	"I've lost count of how many times this has happened just this month. <origin>, <planet>, I wonder what's the difference! At this point, I don't know why no one's started to regularly meet with <planet>'s postmaster to exchange all this wrongly-posted mail. Alright, here's a word of advice: I highly recommend you take a picture of the incorrect address. I've had plenty of captains get put on blast 'cause of an error the sender made. Not your fault it got addressed to the wrong planet."`
+			`	You thank the clerk and take the package back to your ship.`
+			choice
+				`	(Take a picture of the incorrect address.)`
+				`	(Don't bother taking a picture.)`
+					accept
+			action
+				set "took picture of quicksilver package"
+			`	You take a few photographs of the address label, just to be sure that you've got a clear picture of the wrong address.`
+				accept
+	on complete
+		payment 17100
+		event "quicksilver mixup 2 available" 90
+		conversation
+			`When you talk to a post officer in <planet>'s spaceport and point out the incorrect address before handing it over, you note that he exhibits the same frustration the clerk on <origin> did regarding the frequent switch-ups. He takes the package, muttering something about off-worlders "not knowing the difference even if it smacked 'em in the face" as he did.`
+			`	Once you return to your ship, you notice that the man who gave you that package had sent you a message. Or, to be more precise, a series of messages. An angry series of messages.`
+			`	"WHAT TOOK YOU SO LONG???" one of them said. "I NEEDED THAT DELIVERED DAYS AGO", says the next one.`
+			branch picture "didn't take picture"
+				has "took picture of quicksilver package"
+
+			label picture
+			`	You send the man the picture you took of his error and point out that it was his mistake that got the package initially got sent to the wrong place, and politely inform him that had he wanted a smoother delivery, then maybe he should've filed a proper job listing and made sure that he placed the correct address.`
+			branch "picture success" "picture failure"
+				random < 50
+
+			label "picture failure"
+			action
+				payment -17100
+			`	He leaves you on read.`
```

Comment:
"on read"? Was this supposed to be "unread"?

```suggestion
			`	He leaves the message unread.`
```

Reply:
It's a joke (although not one I would have made). You can leave a message unread and apps like WhatsApp will say that you've seen and read the message, but you haven't replied, which is meant to be disrespectful or something.

Reply:
Then let's make it clearer in another way:

```suggestion
			`	He doesn't reply, though you can see that he read the message..`
```

Reply:
The joke explicitly is leaving someone "on read", so this suggestion does take the joke out of it. I don't mind the change, but I'll talk with Improvise who wrote the story whether they would like this suggestion or not.

Reply:
Maybe my age is showing, but "leaving someone on read" is slang I've never heard before. I understand what it means now, but for someone of my vintage, it read like a typo. :)

Reply:
That suggestion very much does remove the joke. 
So I'd suggest either leaving it as it is, and keeping the humor in, or rewriting it entirely to not be a joke at all. 

It looks like "leave on read" has been slang since ~2019, so I would personally not have any concerns with leaving it as it is. It's humorous.

Reply:
What's the joke? That he reads your message doesn't reply to it? Can someone explain what the joke is?

Reply:
The joke here is that the client in question was upset at you, and was sending you a string of angry messages over something that was ultimately their own fault. 
If you point out to them that it was their own fault, and not yours, they suddenly go silent and "leave you on read". 

The sudden shift in dynamic between you and your client is what is supposed to be funny, and that sudden shift relies on the relatively short punchline. 

If you reword the punchline to something that still technically means the same thing, but says something different, it doesn't have the impact required to function as a punchline and the build-up and payoff of the joke both fall flat. 

So if we want to keep the joke, we should keep it as it is, if the slang isn't acceptable, then (in my opinion) it should just be rewritten to not include the joke at all, as no joke is better than a joke that falls flat.

Reply:
I hate to be that guy, but what you described isn't a joke. It may be intended to be _funny_, but it's not a _joke_. All jokes are (or should be) funny, but not everything that's funny is a joke.

And I didn't find it funny due to the use of unfamiliar slang causing me to interpret the final text as a typo. In general this game doesn't use a lot of modern slang in descriptions, and I think it would be best to continue that trend here to preserve clarity. If that requires re-doing some of the text to preserve the humor, so be it.

I've said my peace on this topic and made my opinion known; I'll let others decide how to proceed now.

Reply:
It's certainly not what i'd call a joke, but it's a very common thing to say, and to leave FL on read fits perfectly with what the space karen would do, so i see no reason to change that whole conversation structure. 

I don't think many, if anybody, would have issue understanding, due to how widespread that term is. It is currently a 7 year old expression (originated in 2016 according to dictionary.com) with a lot of staying power. It isn't slang as much a a common everyday expression - newer and less used words have been included in dictionaries. Hell, there's references to furries and fursuits in Endless Sky, which, while have existed before, have become commonly known as what they are referenced as in game more recently, as well, if i am not mistaken. I mean, my autocorrect is tagging both "furries" and "fursuits" as words that don't exist, and i find it much more likely for a player not to know of them than to not know of this expression. I don't see an issue with using "on read" at all.

If we must change it, then I'd write it as `He doesn't respond.`. It would keep the fitting structure. But these are my two cents on how we should proceed and why i think that!

Reply:
Well, I can't tell you what slang will be like in 1000 years' time, and Improvise hasn't got back to me yet. I'll change it to `He doesn't respond`.